L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.cc

```diff
@@ -331,6 +347,12 @@ bool L1TCaloLayer1FetchLUTs(
     }
   }
 
+  // Make HCal FB LUT
+  for (uint32_t etaBin = 0; etaBin < 28; etaBin++) {
```

Comment:
This "28" enters already a few times: it could be made a named `constexpr` then

Reply:
this should probably be used https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TCalorimeter/interface/CaloTools.h#L38

Reply:
Replaced with constexpr "nCalEtaBins".